master/internal/user/postgres_users.go

```diff
@@ -341,7 +350,7 @@ func ByID(ctx context.Context, userID model.UserID) (*model.FullUser, error) {
 		Join("LEFT OUTER JOIN agent_user_groups h ON u.id = h.user_id").
 		Where("u.id = ?", userID).Scan(ctx, &fu)
 	if err != nil {
-		if errors.Cause(err) == sql.ErrNoRows {
```

Comment:
we should ban ```errors.Cause``` from the codebase through linter

Well I guess we should ban "github.com.../errors" too but errors.Cause would be an easy ban and string replace